src/shared/containers/app.jsx

```diff
@@ -245,8 +245,8 @@ class App extends React.Component {
               {toolbox}
               <UserBox store={this.props.store} />
             </ToolbarRow>
-          )}
-        </Toolbar>
+          </Toolbar>
+        )}
         <Drawer
```

Comment:
The drawer could be put in above condition

Reply:
if `displayToolbar` is set to `false` I also remove the `Drawer` ? Is it not too much ? We could want to display it another way than from the hamburger in the toolbar.

Reply:
That why I created an issue with isFullScreen: true, to remove both ;-)

Reply:
Where is this issue with the fullscreen ? Indeed calling it fullscreen is maybe a better idea and then we could hide Toolbar and Drawe

Reply:
https://github.com/Zoapp/front/issues/63

Reply:
ok great I will use this parameter, I didn't saw this issue before.